### ISSUE TITLE
Site data 

### ISSUE THREAD

**Issue by MomosX** [CJNE/ha-sunspec#139]
Hello and thank you for this integration !!!

I have 2 Fronius Symo connected in daisy chain and a Fronius smartmeter.  I set up this integration 3 times, for slave 1 and 2 as the inverters and 240 for the meter.
Data is coming through and all seems good.

However i am struggling to find out how to get the Site data.  Fronius modbus docs specify that
"The following registers can be used to query power and energy data from all inverters
connected to this Fronius Datamanager via Fronius Solar Net.
These values are displayed in Watt (W) or Watt hours (Wh) and do not require scale factors.
- F_Site_Power(500–501): Power
- F_Site_Energy_Day(502–505): Daily Energy
- F_Site_Energy_Year(506–509): Yearly Energy
- F_Site_Energy_Total(510–513): Total energy of the entire system."

This data is available when doing a http call to the meter: "http://ipaddress/solar_api/v1/GetPowerFlowRealtimeData.fcgi". The response contains: 
![image](https://user-images.githubusercontent.com/61582845/192129570-df66ec95-9c5a-4f0b-87ff-915e4297cb5f.png)

Basically what i am trying to get is the Site Power (P_Load)  - aka the total power in Watts for the house and the P_Grid - power from Grid

Is this possible please ? 

Thank you

**Comment by MomosX** [comment]
I figured it out :) just needed to do some calculations to extract that data from the existing one.

Cheers :)
